### ISSUE TITLE
Possible to swipe cards below the card

### ISSUE THREAD

**Issue by michalpawlowski34** [webraptor/react-native-deck-swiper#120]
I am struggling with a bug where it is possible to use gestures for Swiper below the Card component all the way to to the bottom of the screen.
It looks like the Swiper takes the whole available space below.
Tested on iOS with Expo.

Code:
` <View className="flex-1 -mx-3 -mt-10">
          <Swiper
            cards={cardsData}
            renderCard={Card}
            verticalSwipe={false}
            stackSize={2}
            animateCardOpacity
          />
        </View>`

Card's given height is 60%. But no matter if I add margins, paddings 0 or apply it to containerStyle or even mess around with cardVerticalMargin prop it is still possible to swipe below the cards.

Any suggestions?

**Comment by Joe-Brar** [comment]
Same issue. Did you fix it?

**Comment by michalpawlowski34** [comment]
Not really. I suspect that it is caused by the Swiper parent that has the flex style assigned to it, but I didn't test it yet. Anyway might be worth giving a try.

**Comment by michalpawlowski34** [comment]
> Same issue. Did you fix it?

Not really. I suspect that it is caused by the Swiper parent that has the flex style assigned to it, but I didn't test it yet. Anyway might be worth giving a try.
